enclave-tls/src/tls_wrappers/internal/etls_tls_wrapper_select.c

```diff
@@ -32,6 +33,23 @@ enclave_tls_err_t etls_tls_wrapper_select(etls_core_context_t *ctx,
 		if (type && strcmp(type, tls_wrappers_ctx[i]->opts->type))
 			continue;
 
+		if (tls_wrappers_ctx[i]->opts->flags) {
```

Comment:
Change to `if (... & TLS_WRAPPER_OPTS_FLAGS_SGX_ENCLAVE)` because more flags may be introduced in future.